texts/epn_paper.md

```diff
@@ -0,0 +1,67 @@
+# Let's talk online
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine changed drastically over the course of the last couple of months due to the lockdowns. You now mostly stay at home, hopefully safe and healthy.
+Your classes are now online, and probably you had to quickly figure out how run them at all.
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite that, you have likely attended a scientific presentation or even gave one yourself.
+The only difference is that it happened online.
+
+<!-- #### Online talks were gaining momentum already before, and for a good reason. -->
+
+Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
+With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
+To some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether.
+Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.
+With all these obstacles out of the way, virtual conferences can be organized much faster and respond to the latest developments.
+They are also more inclusive and reach a broader audience especially if the talks are recorded.
+
+<!-- #### We decided to develop online conferences and established VSF. -->
+
+After finding each other as like-minded individuals with an urge to improve the present situation, we started gathering in the late summer of 2019 to explore our options. 
```

Comment:
Here I think "we" comes out of nowhere, and the connection to the previous paragraph is unclear.

How about
```suggestion
The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researhers.
We started exploring the opportunities and challenges of online conferences in late Summer 2019—well before the COVID pandemic.
We decided to keep our organization open for anyone to join, and all of our documentation and notes public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
```

Reply:
I agree that the 'we' needs a bit more clarification. I like your suggestion too.